docs/guides/templates.md

```diff
@@ -34,14 +34,16 @@ npx create-remix@latest --template remix-run/remix/templates/express
 npx create-remix@latest --template remix-run/remix/templates/remix
 npx create-remix@latest --template remix-run/remix/templates/remix-javascript
 
+## Deno
+deno -A npm:create-remix@latest --template remix-run/remix/templates/deno
```

Comment:
alternatively this could be `npx create-remix@latest --template remix-run/remix/templates/deno --package-manager deno`, but i think using deno here makes more sense as it avoids needing another runtime + package manager just to scaffold.